1.7/overview/security.md

```diff
@@ -69,191 +69,4 @@ Authenticated users are authorized to perform arbitrary actions in their
 cluster. That is, there is currently no fine-grained access control in DC/OS
 besides having access or not having access to services.
 
-## Authentication Flow
```

Comment:
It is good to get this split out, but the article still doesn't give me a good overview of what's going on. @alberts any suggestions?